src/profiler/profiler.c

```diff
@@ -249,8 +295,9 @@ void xdebug_profiler_deinit()
 
 	fprintf(
 		XG_PROF(profile_file),
-		"summary: %lu %zd\n\n",
-		NANOTIME_SCALE_10NS(xdebug_get_nanotime() - XG_PROF(profiler_start_nanotime)),
+		"summary: %lu+%lu %zd\n\n",
```

Comment:
do we want/can we (because it it parsed somewhere?) to display total time as sum of php time and profiler time?

Reply:
This is part of the callgrind spec, and the format can't change.

Reply:
It parsed for me by `qcachegrind`, I see, good point, the only option would be to add completely different `cost` (callgrind spec), but xdebug overhead is not so much important for the end user/developer... I will keep displaying only the real time spent in the php.